src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java

```diff
@@ -17,13 +17,18 @@
 
     @NotNull
     @NotEmpty
+    private Boolean sync;
```

Comment:
succeeded it's good ! I do not understand the second comment ... can you explain me more please